SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Storage/StorageAdapter.cs

```diff
@@ -92,54 +91,66 @@ private object GetColumnValue(sqlite3_stmt stmt, int index)
             return null;
         }
 
-        private int ExecuteNonSelectionSqlQuery(string query, IList<object> args = null)
+        private void ExecuteNonSelectionSqlQuery(string query, IList<object> args = null)
         {
             var db = _db ?? throw new StorageException("The database wasn't initialized.");
             var result = raw.sqlite3_prepare_v2(db, query, out var stmt);
             if (result != raw.SQLITE_OK)
             {
                 throw ToStorageException(result, "Failed to prepare SQL query");
             }
-            BindParameters(stmt, args);
-            result = raw.sqlite3_step(stmt);
-            if (result != raw.SQLITE_DONE)
+            try
+            {
+                BindParameters(stmt, args);
+                result = raw.sqlite3_step(stmt);
+                if (result != raw.SQLITE_DONE)
+                {
+                    throw ToStorageException(result, "Failed to run query");
```

Comment:
```suggestion
                    throw ToStorageException(result, $"Failed to run query, result={result}");
```

Reply:
No, it will be added in `ToStorageException`